storage/factory/common.go

```diff
@@ -48,9 +50,40 @@ func GetBloomFromConfig(cfg config.BloomFilterConfig) storageUnit.BloomConfig {
 	}
 }
 
+// loadEpochStartRound will return the epoch start round from the bootstrap unit
+func loadEpochStartRound(
+	shardID uint32,
+	key []byte,
+	storer storage.Storer,
+) (uint64, error) {
+	trigInternalKey := append([]byte(core.TriggerRegistryKeyPrefix), key...)
```

Comment:
this is a super specific function, should not stay in common, should be part of bootstrapperdataprovider

Reply:
done.